### DISCUSSION TITLE
Issue getting plain text to display when inserting nodes

### DISCUSSION THREAD

**Discussion by bchr0n** [chipsenkbeil/org-roam.nvim#58] [Q&A] [answered]
When I create a new node, it always shows the id of the node, never the plain-text title. I followed along in the video on your YouTube and even when I go back to the start of the node like you do and separate the node it still stays in that style. The notes are created inside of my org-roam directory and are in no subdirectory. I am very new to Linux and Neovim and even newer to Org-Mode so I don't even know where to begin to figure out what the issue is. Any help is appreciated, thank you!

Below is a picture of behavior

![Screenshot from 2024-07-31 13-30-42](https://github.com/user-attachments/assets/0c5caf26-aa54-46c4-a9c0-cb0568723fe9)

Below is also my org-roam init file

![Screenshot from 2024-07-31 13-32-00](https://github.com/user-attachments/assets/f649fa8a-578a-450d-bab2-b0f142212bd3)

**Comment by chipsenkbeil** [comment, marked as answer]
> it always shows the id of the node, never the plain-text title

Do you mean that the link shows up like `[[id:abcd-1234-...|My Note]]` instead of `My Note`?

If so, have you checked out [orgmode.nvim's documentation about link concealment](https://github.com/nvim-orgmode/orgmode?tab=readme-ov-file#links-are-not-concealed)?

**Reply by bchr0n** [reply]
Wow I don't know how I missed that, I looked through the documentation but didn't think to actually look the README itself. Thank you for your help!
